deploy/charts/external-secrets/tests/__snapshot__/cert_controller_test.yaml.snap

```diff
@@ -41,7 +41,7 @@ should match snapshot of default values:
                 - --loglevel=info
                 - --zap-time-encoding=epoch
                 - --enable-partial-cache=true
-              image: oci.external-secrets.io/external-secrets/external-secrets:v0.20.2
+              image: ghcr.io/external-secrets/external-secrets:v0.20.2
```

Comment:
the version looks weird - but not related to these changes. Just for the record (we are also discussing on slack.)